region/crateria/east/West Ocean.json

```diff
@@ -1249,6 +1250,28 @@
         "h_canCrystalFlash"
       ]
     },
+    {
+      "link": [12, 13],
+      "name": "West Ocean - Enemy Stuck Moonfall into the Ocean",
+      "notable": true,
+      "requires": [
+        "canEnemyStuckMoonfall",
+        "canTrickyUseFrozenEnemies",
+        "Morph",
+        {"or": [
+          "Bombs",
+          "SpringBall"
```

Comment:
Looks like you can freeze the Zeb and still get back to it in time while using a Power Bomb to get through the morph tunnel